README.md

```diff
@@ -10,16 +10,16 @@
 
 Jagrati is an initiative by the students of **IIITDM Jabalpur** to provide free and quality education to the poor and under-privileged children of villages surrounding our institute. 
 
-Currently, we have adopted 5 villages in the vicinity of our institute, namely, Gadheri, Amanala, Chanditola, Mehgawan and Suarkol where we *donate education* to around 100 children of classes 1 through 10. Apart from providing basic education to the students in regular classes, **we also prepare the students of 4th and 5th grade for the prestigious Navodaya Vidyalaya** and other similar government-funded institutions which provides quality education to students, completely free of cost.
+Currently, we have adopted 5 villages in the vicinity of our institute, namely, Gadheri, Amanala, Chanditola, Mehgawan and Suarkol where we *provide education* to around 100 children of classes 1 through 10. Apart from providing basic education to the students in regular classes, **we also prepare the students of 4th and 5th grade for the prestigious Navodaya Vidyalaya** and other similar government-funded institutions which provides quality education to students, completely free of cost.
 
 Apart from teaching the children, we do many other activities like **organizing Blood Donation Camps, Tree Plantation Drives, Cloth Donation, Stationery Distribution, Campaigns to spread awareness in villages, etc.**
 
 **Our Achievements:**
-- 1 student selected in Jawahar Navodaya Vidyalaya in 2018.
-- 1 student selected in Jawahar Navodaya Vidyalaya in 2019.
-- 2 students selected in Jawahar Navodaya Vidyalaya in 2020.
-- 4 students cleared entrance examination for Eklavya Model Residential School (EMRS) in 2020.
-- 4 students cleared entrance examination for Gyanodaya Vidyalaya in 2020.
+- 1 student was selected in Jawahar Navodaya Vidyalaya in 2018.
+- 1 student was selected in Jawahar Navodaya Vidyalaya in 2019.
+- 2 students were selected in Jawahar Navodaya Vidyalaya in 2020.
```

Comment:
One more small change, I think in place of `in` in above three lines, `for` should be present.